### PR TITLE
GH-2

### PR DESCRIPTION
**Pull request by mwi-dtroyer**
Used localStorage to show whether or not a user is logged in

### CODE REVIEW

File: src/ChecklistDojo/ClientApp/src/components/NavMenu.js [hunk 1]
```diff
@@ -30,53 +30,84 @@ export class NavMenu extends Component {
...
-                    Settings
-                  </NavLink>
-                </NavItem>
-              </ul>
-            </Collapse>
-          </Container>
-        </Navbar>
-      </header>
-    );
+    if (localStorage.getItem("isLoggedIn", "true")) {
```
Comment (mwi-pharju) [inline src/ChecklistDojo/ClientApp/src/components/NavMenu.js]: The fact that this change causes the app to show the correct button seems like more of an unintended side-effect than a direct solution, especially when this doesn't even touch the home component. 

I think this `if` statement is triggering another rendering cycle, causing the home component to re-check the logged in flag and get a different result than last time, due to the request being fast enough that it was able to finish between the first render and the re-render caused by this if/else check.

In short, I don't think this is the approach we want to take.
